### PR TITLE
Add `cluster` param to eql track for CCS

### PR DESCRIPTION
**Pull request by matriv**
Add a new param: `cluster` which can be set to indicate
which configured remote cluster to hit when running the EQL
benchmarks.

### CODE REVIEW

**Comment by matriv** [comment]
PS. This is just for the future, used it only manually to execute CCS related benchmarks.

File: eql/track.py [hunk 1]
```diff
@@ -4,8 +4,11 @@
 async def eql(es, params):
     logger = logging.getLogger(__name__)
     logger.info("Provided params [%s].", params)
+    cluster = ""
+    if params.get("cluster"):
+        cluster = params.get("cluster") + ":"
```
Comment (dliappis) [inline eql/track.py:9]: The three lines above can be replaced with:

```
if cluster := params.get("cluster", ""):
    cluster += ":"
```

**Review by dliappis — APPROVED** [review]
LGTM. I left a suggestion for something simpler.

**Comment by matriv** [comment]
Backported to `7.14`, `7.12` & `7.2`.

**Comment by dliappis** [comment]
Thanks for all the work here!

**Pull request merged**